faqs/galaxy/dataset_tool_erorrs_metadata.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Tool Errors related to Metadata
+area: datasets   
+description:  Finding and Correcting Metadata
+layout: faq         
+---
+
+Tools can error when the wrong dataset attributes (metadata) are assigned. Some of these wrong assignments may be: 
+ - Tool outputs, which are automatically assigned without user action
+ - Incorrect autodetection of datatypes, which need manual modification
+ - Undetected attributes, which require user action (example: assigning database to newly uploaded data)
+
+How to notice missing Dataset Metadata:
+- Dataset will not be downloaded when using the disk icon.
```

Comment:
you could use icons here (see https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/faqs/icons_list.html).

Reply:
Done.